src/pages/map/map.spec.ts

```diff
@@ -34,11 +37,19 @@ describe('MapPage', function () {
       getCurrentPosition: stub().returns(geolocationDeferred.promise)
     };
 
+    popoverMock = {
+      present: stub().returns(new Deferred().promise)
+    };
+    popoverCtrlMock = {
+      create: stub().returns(popoverMock)
+    };
+
     navControllerMock = {};
 
     TestBed.configureTestingModule({
       declarations: [
-        MapPage
+        MapPage,
+        MockComponent({ template: `<h1>Toto</h1>` })
```

Comment:
I don't believe this `MockComponent` is used in the test, since the popover controller is mocked anyway. Unless it's already used in future branches, it should be removed along with the new `ng2-mock-component` dependency.